### DISCUSSION TITLE
fatal: Could not resolve HEAD to a revision installing node@16

### DISCUSSION THREAD

**Discussion by eleon01** [Homebrew/discussions#3619] [Casks]
Hi, I've been trying to brew install node@16 in an Ubuntu vm, but I get that fatal error and a warning:
Warning: No available formula with the name "node@16".
Any suggestions on how to fix it? Thanks.

**Comment by carlocab** [comment]
What is the output of `brew config` and `brew doctor`?

**Reply by eleon01** [reply]
Omg it worked 😭 thanks carlo.
